third_party/go-tun2socks/METADATA

```diff
@@ -0,0 +1,18 @@
+name: "go-tun2socks"
```

Comment:
This repository doesn't depend on go-tun2socks directly. This directory should be outline-go-tun2socks to proper reflect that. The comments on this file should be updated.

Since this is a large PR, it's ok to rename on a follow-up PR

Reply:
Done in 3e73f85.